apps/server/src/modules/authentication/uc/account.uc.ts

```diff
@@ -174,13 +176,15 @@ export class AccountUc {
 	 * @param params account details
 	 */
 	async updateMyAccount(currentUserId: EntityId, params: PatchMyAccountParams) {
-		let account: Account;
+		let user: User;
 		try {
-			account = await this.accountRepo.findByUserId(currentUserId);
+			user = await this.userRepo.findById(currentUserId, true);
 		} catch (err) {
-			throw new EntityNotFoundError('Account');
+			throw new EntityNotFoundError('User');
```

Comment:
Better and more refactoring friendly way the get the name of an entity is 'User.name'.

Reply:
Done.